components/classroom/styles.css

```diff
@@ -0,0 +1,145 @@
+* {
+    box-sizing: border-box;
+}
+
+*::before, *::after {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    background-color: rgb(240, 241, 241);
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
+    overflow-x: hidden;
+}
+
+.header {
+    display: grid;
+    padding: 30px;
+    text-align: center;
+    background: #516059; 
+    color: white;
+}
+
+.header h1 {
+    font-size: 30px;
+}
+
+.page-container h2 {
+    color: #516059;
+}
+
+.navbar {
+    font-size: clamp(1rem, 2vh, 2rem);
+    background-color: #516059;
+}
+
+.navbar ul {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    padding-bottom: 0.5rem;
+}
+
+li a {
+    display: block;
+    color: #fff;
+    padding: 10px 10px;
+    text-decoration: none;
+}
+
+li a:hover {
+    background-color: #ffffff;
+    color: #516059;
+    border-radius: 10px;
+}
+
+.page-container {
+    text-align: center;
+}
+
+.input-container {
+    padding-top: 1rem;
+}
+
+.input-form {
+    padding-bottom: 0.5rem;
+}
+
+footer {
+    padding: 10px;
+    text-align: center;
+    background: #516059; 
+    color: white;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+}
+
+/* Hamburger icon */
+.hamburger-menu {
+    display: none;
+    font-size: 25px;
+    cursor: pointer;
+    position: absolute;
+    top: 20px;
```

Comment:
super nitpicky but would look better if it was aligned with the website header! so top 30px instead of 20px